src/analysis/fourier/riemann_lebesgue_lemma.lean

```diff
@@ -0,0 +1,152 @@
+/-
+Copyright (c) 2022 David Loeffler. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: David Loeffler
+-/
+import measure_theory.function.lp_space
+import measure_theory.function.continuous_map_dense
+import measure_theory.integral.interval_integral
+import measure_theory.integral.integral_eq_improper
+import measure_theory.group.integration
+import topology.continuous_function.zero_at_infty
+
+/-!
+# The Riemann-Lebesgue Lemma
+
+In this file we prove a weak form of the Riemann-Lebesgue lemma, stating that for any
+compactly-supported continuous function `f` on `ℝ` (valued in some complete normed space `E`), the
+integral
+
+`∫ (x : ℝ), exp (I * t * x) • f x`
+
+tends to zero as `t → ∞`. (The actual lemma is that this holds for all `L¹` functions `f`, which
+follows from the result proved here together with the fact that continuous, compactly-supported
+functions are dense in `L¹(ℝ)`, which will be proved in a future iteration.)
+
+## Main results
+
+- `tendsto_integral_mul_exp_at_top_of_continuous_compact_support`: the Riemann-Lebesgue lemma for
+  continuous compactly-supported functions on `ℝ`.
+-/
+
+open measure_theory filter complex set
+open_locale filter topology real ennreal
+
+section continuous_compact_support
+
+variables {E : Type*} [normed_add_comm_group E] [normed_space ℂ E] {f : ℝ → E}
+
+lemma norm_exp_mul_mul_I (t u : ℝ) : ‖exp (I * t * u)‖ = 1 :=
```

Comment:
Can you identify a file where it belongs to naturally, and put it there? (probably `data.complex.exponential`, unless the norm is not defined yet there).